build.gradle

```diff
@@ -343,42 +332,31 @@ task setupEclipse {
     group "Setup"
     dependsOn eclipseProject, createEclipseLaunches, resetEclipseDeps
 }
+*/
```

Comment:
This huge block is commented out. Is that intended?

Reply:
> Support for both IDEs are totally removed here; they need to be restored (independently) as the follow-ups of this PR.